lib/src/components/NxFilterInput/NxFilterInput.scss

```
@@ -9,6 +9,10 @@
 
 $nx-filter-input-height: 20px;
 
+.nx-filter-input, .nx-filter-input * {
+  box-sizing: content-box;
+}
+
 .nx-filter-input {
   @include container-horizontal;
 
```

Selecting more precisely:

```suggestion
.nx-filter-input {
  @include container-horizontal;

  > * {
    box-sizing: content-box;
  }
```